src/renderStream.ts

```diff
@@ -0,0 +1,43 @@
+import * as fs from 'fs';
+import * as util from 'util';
+
+import {renderSync} from './render';
+import {StyleConfig} from './types';
+import {validateRenderData} from './validate';
+
+const readAsync = util.promisify(fs.readFile);
+
+/**
+ * Take JSON input from stdin and generate and output a graph to stdout
+ */
+export async function renderStream(styles: StyleConfig) {
+  const json = await readAsync(0, {encoding: 'utf8'});
```

Comment:
There's a global `process.stdin.fd` which can point to the stdin fd depending the system. But it should be 0 in most cases.

Reply:
that's much more clear for documentation reasons though. Thanks!